sonar-project.properties

```diff
@@ -0,0 +1,10 @@
+sonar-project.properties
+sonar.organization=fedora-infra
+sonar.projectKey=fedora-infra/noggin
+sonar.sources=.
```

Comment:
shouldn't this be `noggin`?

Reply:
I'm sorry, I'm not sure what you mean by "this"

Reply:
The `sonar.sources` value should point at the directory where the sources are. This should probably be `noggin` since that's where our code lives, I don't think we need Sonarqube to analyze the ansible files, default config files, etc. No?

Reply:
You're right, that's changed to noggin now